server.js

```diff
@@ -24,9 +17,116 @@ const app = express();
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
+const Animal = mongoose.model("Animal", {
+  name: String,
+  scientificName: String,
+  doesItFart: Boolean,
+  doesItMaybeFart: Boolean,
+  descriptionExists: Boolean,
+  descriptionNotes: String,
+  paperLink: String,
+  verifiedBy: String,
+});
```

Comment:
Really fun theme you came up with, great work to use another dataset then the ones provided.